plugins/guests/redhat/cap/configure_networks.rb

```diff
@@ -48,6 +48,23 @@ def self.configure_networks(machine, networks)
               raise Vagrant::Errors::NetworkManagerNotInstalled, device: network[:device]
             end
 
+            # Render static route file if present
+            if network.key?(:route) && network[:route] != nil
```

Comment:
No need to explicitly check nil, just checking `network[:route]` will be good enough

```suggestion
            if network.key?(:route) && network[:route]
```